### ISSUE TITLE
MD038 shouldn't trigger when string starts OR ends with a backtick

### ISSUE THREAD

**Issue by jack1142** [DavidAnson/markdownlint#386]
Here's an example where this would be useful:
```
`` Ctrl+` ``
```
Right now it triggers MD038 error, but it's the only way to write "Ctrl+\`" without a trailing space ending up the inline code.

**Comment by DavidAnson** [comment]
This pattern explicitly should not trigger MD038 as described in the documentation: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md038

Here is an example of that text not reporting an issue: https://dlaa.me/markdownlint/#%25m%23%20Issue%20386%0A%0AText%20%60%60%20Ctrl%2B%60%20%60%60%20text.%0A

**Comment by jack1142** [comment]
Yeah... That's a bit embarrassing, I forgot to update before reporting. Sorry!
